backend/booking/serializers.py

```diff
@@ -159,6 +159,7 @@ class Meta:
         model = Organization
         fields = '__all__'
 
+from django.db.models import Q
```

Comment:
Why are we importing in middle of the file?

Reply:
will move it now